README.md

```diff
@@ -1 +1,84 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+- [Setup](#setup)
+- [Run tests](#run-tests)
+- [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# Portfolio-setup-and-mobile-first
```

Comment:
- [OPTIONAL] It would look nice in your readme if you can keep the title same as your application title for consistency purposes. So for example, in your case, you can change it to Portfolio 👍🏽 💯